### PR TITLE
Pass ergm.control to within-block estimation

### LINKED ISSUE #28: Pass arguments to within-block ergm
Except for the method, can't pass arguments to the within-coefficients estimation ergm.
https://github.com/sansan-inc/lighthergm/blob/32dc3ea01e9f88151143ca5344f20134baf1afa0/lighthergm/R/estimate_within_params.R#L179

For example, it could be useful to pass an `ergm.control` object to that function when externalities are used.

### PR DESCRIPTION
**Pull request by justinian336**
Closes #28 

You can now pass a **control** argument to `lighthergm::hergm`, which must be of the type ergm::control.ergm.
It's possible to just create another named argument, but I thought that putting inside the variadic arguments is more extensible: in the future we may want to pass all the options that the `ergm::ergm` function accepts, which could be done in the form of varargs.

### CODE REVIEW

File: lighthergm/R/estimate_within_params.R [hunk 1]
```diff
@@ -177,11 +179,18 @@ estimate_within_params <-
 
     # Estimate the within-block parameters
     # The default estimation method is "MPLE", but you can select "MLE" if you like.
+    if(is.null(varargs$control)){
+      control <- ergm::control.ergm()
+    } else {
+      control <- varargs$control
+    }
```
Comment (komatsuna4747) [inline lighthergm/R/estimate_within_params.R]: These could be written in one line, though I'm not sure which is better.
```suggestion
control <- switch(is.null(varargs$control) %>% as.character, "TRUE" = ergm::control.ergm(), "FALSE" = varargs$control)
```

**Review by komatsuna4747 — APPROVED** [review]
LGTM 👍   I verified that the test passed. I just left one comment.

File: lighthergm/R/estimate_within_params.R (hunk 1 above)
Comment (justinian336) [reply lighthergm/R/estimate_within_params.R]: @komatsuna4747 
Thanks for the comment. I'd rather not use a `switch` statement for a boolean. I just discovered the `%||%` infix operator. The following one-liner should achieve the same, what do you think?
```R
control <- rlang::`%||%`(varargs$control, ergm::control.ergm())
```

Comment (komatsuna4747) [reply lighthergm/R/estimate_within_params.R]: @justinian336 
Wow, this is what I have been looking for. I'd prefer `%||%` to the if-else statement in terms of simplicity. But would it be kind to write a comment on what the operator does?

If you use the `%||%` operator, I'd suggest the following:
```r
#' @importFrom rlang %||%

control <- varargs$control %||% ergm::control.ergm()
```

Comment (justinian336) [reply lighthergm/R/estimate_within_params.R]: @komatsuna4747 thanks for the comment. I made the changes you suggested. Will merge as soon as the checks run successfully.

**Pull request merged**